app/grandchallenge/profiles/forms.py

```diff
@@ -75,4 +82,7 @@ def signup(self, request, user):
         user_profile.department = self.cleaned_data["department"]
         user_profile.country = self.cleaned_data["country"]
         user_profile.website = self.cleaned_data["website"]
+        user_profile.organization_display = self.cleaned_data[
+            "organization_display"
+        ]
```

Comment:
We shouldn't show this when the user signs up, only have it as an option when they are editing their profile later. Otherwise, this form gets too messy, and when they are new they do not need to know about organisations.

Reply:
Same behavior as `mugshot`, you will also need to pop it from the fields in L34/L38.

Reply:
ok, now that you meantion the sign-up, I get an error message when I try to sign-up `IntegrityError: null value in column "organizations" violates not-null constraint`. I can't figure out how to allow organizations to be empty...

Reply:
I don't really understand, I cannot see where an `organizations` column would be coming from with anything in this changeset.

Reply:
Is the default set correctly?

Reply:
Yeah, I didn't add an organizations field anywhere and so also didn't set a default, that's why I'm a bit confused... the only place where I define organizations is in views.py `organizations = Organization.objects.filter(members_group__in=profile_groups`

Reply:
never mind, it works